scan/src/components/ChainInfoHighlights.re

```diff
@@ -1,15 +1,16 @@
 module Styles = {
   open Css;
 
-  let card =
+  let card = isMobile =>
```

Comment:
It's no need to use `isMobile` anymore because there is the `Media.mobile` already.